lib/blocks/blockreader.go

```diff
@@ -248,6 +275,14 @@ func (br *Reader) doTheThingPatternMatch(fs afero.Fs, filename string, stdin io.
 		}
 	}
 
+	var format blockFormat
```

Comment:
given this is the "file", or more abstractly "input text" format, not the block format could we make these
```suggestion
	var format textFormat
```